### ISSUE TITLE
Blacksmith makes normal items

### ISSUE THREAD

**Issue by RensSwag2001** [TangoTek/TekTopia-Community#231]
**Describe the bug**
My blacksmith gets villager diamonds and makes regular armor of them..

**Expected behavior**
Well I expected to get villager items when i give them villager items.

**Screenshots**
![2019-05-31_21 05 41](https://user-images.githubusercontent.com/51168774/58728560-7b403400-83e7-11e9-8c94-fc4baabbfed4.png)

** Server/Location **
Server 1
Towh hall token is placed at -1331 66 -8808

Might be a duplicate of #122

**Comment by JohnyDL** [comment]
if it used just 1 diamond that was not a villager item then the item it crafts will not be a villager item either this is intended. They should also try to use up the none villager items first... make sure they aren't accidentally getting them elsewhere

**Comment by RensSwag2001** [comment]
I store my diamonds in a chest thats 2-high, can villagers pick up items from there?

**Comment by JohnyDL** [comment]
Yup, they should be able to grab from any chest they can path to, and if they couldn't path to that chest then you'd have bigger problems, check your stash or put your things outside the community storage room
